modules/ROOT/pages/workday/workday-connector-studio.adoc

```diff
@@ -32,35 +31,35 @@ To configure a connector in Anypoint Studio:
 
 == Configure an Input Source
 
-Configure an input source for the connector such as an operation for
-the Workday connector, *HTTP Listener*, or *Scheduler*.
-
-You can put the following operation for the connector in
-the *Source* area of the Studio canvas:
+Configure an input source for the connector.  Possible input source operations are:
 
-New objects trigger
+* *On New Objects* - Start your app when an object is created in Workday. 
+* *HTTP Listener* - Start your app using a browser or a command such as `curl`.
+* *Scheduler* - Start your app at timed intervals.
 
-. Click the Trigger card and select *Workday Connector*: +
-A list of object types that you can use to trigger Mule applications with the Workday connector is displayed. +
-In the Workday connector, the input source trigger is named *New Objects Trigger*.  +
-//. Specify the Workday https://community.workday.com/sites/default/files/file-hosting/productionapi/operations/index.html[Operation].
-. Select *Click here to set it up* to set the required fields for the *User Name*, *Password*,
-and *Tenant Name*. This information is available from your Workday account.
+In Studio, drag the input operation you choose from the *Mule Palette* to the *Source* panel in the canvas. 
 
-== Configure the Connector
+The required fields for the *On New Objects* input source are:
 
 [%header,cols="30s,70a"]
 |===
-|Field |Description
-|Configuration Name |The name of the configuration.
-|Connection |The connection associated with the configuration.
+|Studio Field |Description
+|Connector configuration |Global Element Properties configuration name.
+|Object type |One of: `ABSENCE_INPUTS`, `ACADEMIC_CONTACTS`, `ALL_CURRENCIES`, `APPLICANTS`, `BASIC_CUSTOMERS`, `CANDIDATES`, `CERTIFICATION_ISSUERS`, `CERTIFICATIONS`, `COMPETENCIES`, `DEGREES`, `DONORS`, `EDUCATION_TEST_RESULTS`, `EDUCATIONAL_INSTITUTION_COURSE`, `HEALTH_CARE_RATES`, `INVESTORS`, `ORGANIZATIONS`, `PAYMENTS`, `PROGRAMS_OF_STUDY`, `STUDENTS`, `STUDENTS_APPLICATIONS`, `WORKERS`
 |Username |The username to log in with.
 |Password |The corresponding password.
 |Tenant Name |The Workday Tenant ID, which usually has the suffix `pt_1` appended to it, such as `acme_pt1`.
-|Host Name |The host name of one of Workday’s Cloud Servers. The default value is: `impl-cc.workday.com`.
 |===
 
-== To Increase Timeout in Studio
+== Configure Other Operations
+
+Each Workday operation has separate fields in Anypoint Studio that may be required. For more information, see 
+the xref:workday/workday-reference.adoc[Connector Reference].
+
+The topic that follows provides additional information about configuring Workday for use in Studio.
```

Comment:
OMIT
Each topic must be independent, particularly for reuse. The place to identify subtopics is in the introduction to the container topic. Do not provide transitions at the ends of topics.